autoeq/constants.py

```diff
@@ -292,4 +292,38 @@
             'max_fc': 10000.0,
         }] * 8
     },
+    'VOICEMEETER_PEQ': {
+        'optimizer': {
+            'min_f': 20,
+            'max_f': 20000,
+            'max_time': None,
+            'min_change_rate': None,
+            'min_std': 0.008,
+        },
+        'filter_defaults': {
+            'min_fc': 20.0,
+            'max_fc': 10000.0,
+            'min_q': 0.18248,
+            'max_q': 6.0,
+            'min_gain': -20,
+            'max_gain': 20
+        },
+        'filters': [{
+            'type': 'LOW_SHELF',
+            'fc': 65.406392,
+            'q': 0.7
+        }, 
+            {'type': 'PEAKING'},
+            {'type': 'PEAKING'},
+            {'type': 'PEAKING'},
+            {'type': 'PEAKING'},
+        {
+            'type': 'HIGH_SHELF',
+            'fc': 16744.036,
```

Comment:
@JuhaTP is the center frequency of the high shelf filter fixed? Here you have both the fixed value and the minimum and maximum configurable values. Typically I've set fixed high shelf to 10 kHz, is that possible in Voicemeeter?

Reply:
Not fixed. Value there is probably suites for test headphones I used. Voicemeeter BANANA allows to set center/cut-off frequency for all filters up to 20kHz at least (I'm on Ubuntu so can't check other but User Manual and their web page images) .